integration_tests/tests/manage/premises.cy.ts

```diff
@@ -94,25 +94,75 @@ context('Premises', () => {
+        const premises = cas1PremisesSummaryFactory.build()
+        cy.task('stubSinglePremises', premises)
+        const placements = cas1SpaceBookingSummaryFactory.buildList(1)
+        cy.task('stubSpaceBookingSummaryList', { premisesId: premises.id, placements, pageSize: 9 })
+
+        // When I visit premises details page
+        const page = PremisesShowPage.visit(premises)
+
+        // And I select to the 'search' tab
+        page.shouldSelectTab('Search for a booking')
+
+        // Then the 'search' tab should be selected
+        page.shouldHaveTabSelected('Search for a booking')
+
+        // And I should see the search form
+        page.shouldShowSearchForm()
+
+        // And I should not see the results list
+        page.shouldNotShowPlacementsResultsTable()
+
+        // When I submit a search using the form
+        page.searchByCrnOrName('Aadland')
+
+        // Then the 'search' tab should be selected
+        page.shouldHaveTabSelected('Search for a booking')
+
+        // And the search form should be populated with my search term
+        page.shouldShowSearchForm('Aadland')
+
+        // And I should see the results
+        page.shouldShowListOfPlacements(placements)
+
+        // When I search for a name that returns no results
+        cy.task('stubSpaceBookingSummaryList', { premisesId: premises.id, placements: [] })
+        page.searchByCrnOrName('No results for this query')
+
+        // Then the 'search' tab should be selected
+        page.shouldHaveTabSelected('Search for a booking')
+
+        // And the search form should be populated with my search term
+        page.shouldShowSearchForm('No results for this query')
+
+        // Then I should see a message that there are no results
+        page.shouldShowNoResults()
+      })
+
+      it('should not show the placements section if space bookings are not enabled for the premises', () => {
         // Given there is a premises in the database that does not support space bookings
         const premises = cas1PremisesSummaryFactory.build({ supportsSpaceBookings: false })
         cy.task('stubSinglePremises', premises)
 
         // When I visit premises details page
         const page = PremisesShowPage.visit(premises)
 
-        // Then I should not see a list of bookings
-        page.shouldNotShowPlacementsList()
+        // Then I should not see the placements section
+        page.shouldNotShowPlacementsSection()
       })
     })
 
-    it('should not show the placements list if the user lacks permission', () => {
+    it('should not show the placements section if the user lacks permission', () => {
       cy.task('reset')
       // Given I am logged in as a user without access to the booking list
       signIn(['future_manager'])
       // Given there is a premises in the database
-      const premises = cas1PremisesSummaryFactory.build()
+      const premises = cas1PremisesSummaryFactory.build({
```

Comment:
The default premisesSummary from the factory has the `supportsSpaceBookings` flag true as I thought should be correct going forward.